include/zephyr/dt-bindings/pinctrl/renesas/pinctrl-rx.h

```diff
@@ -0,0 +1,10 @@
+/*
+ * Copyright (c) 2024 Renesas Electronics Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_INCLUDE_DT_BINDINGS_PINCTRL_PINCTRL_RX_H_
```

Comment:
why is this empty file needed?

Reply:
This file is removed in the commit thank you